lighthouse-core/gather/driver.js

```diff
@@ -1445,16 +1445,15 @@ class Driver {
   async clearDataForOrigin(url) {
     const origin = new URL(url).origin;
 
-    // Clear all types of storage except cookies, so the user isn't logged out.
+    // Clear some types of storage.
+    // Cookies are not cleared, so the user isn't logged out.
+    // indexeddb, websql, and localstorage are not cleared to prevent loss of potentially important data.
     //   https://chromedevtools.github.io/debugger-protocol-viewer/tot/Storage/#type-StorageType
     const typesToClear = [
       'appcache',
       // 'cookies',
       'file_systems',
-      'indexeddb',
```

Comment:
do we have an existing test that some storage types aren't cleared that these can be added to so we get some assurance we don't accidentally regress at some point?

Reply:
No, but I think I can add this